include/dts-environment.sh

```diff
@@ -117,6 +121,9 @@ BASE_CLOUDSEND_LOGS_URL="39d4biH4SkXD8Zm"
 BASE_CLOUDSEND_PASSWORD="1{\[\k6G"
 DEPLOY_REPORT="false"
 
+BASE_DTS_LOGS_URL="xjBCYbzFdyq3WLt"
+DTS_LOGS_PASSWORD="/w\J&<y1"
```

Comment:
Can you explained why we use hardcoded plaintext password here?

Reply:
Because there's no reason for it to be secret, just like `BASE_CLOUDSEND_PASSWORD`

Reply:
You didn't really explain why.
From what I can see when reviewing this is that you put plaintext password that's visible to everyone (public repository).

Reply:
This is just a password to the folder where the logs will be sent. The share only allows uploading files to the folder - you can't even view the existing files, so this is safe.